From 06e4a922d6a46612a3ffb179594498804a77f222 Mon Sep 17 00:00:00 2001
From: pelya <x.pelya.x@gmail.com>
Date: Sun, 16 Mar 2014 21:29:46 +0200
Subject: [PATCH 0073/1053] Renamed buttons on tablet toolbar, and removed
 "Move" button - it is not needed with left_mouse_btn_scrolling

---
 src/lang/english.txt | 4 ++--
 src/toolbar_gui.cpp  | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/lang/english.txt b/src/lang/english.txt
index ad4759305a..231d386712 100644
--- a/src/lang/english.txt
+++ b/src/lang/english.txt
@@ -2446,9 +2446,9 @@ STR_LANDSCAPING_LEVEL_LAND_TOOLTIP                              :{BLACK}Level an
 STR_LANDSCAPING_TOOLTIP_PURCHASE_LAND                           :{BLACK}Purchase land for future use. Shift toggles building/showing cost estimate
 
 # Tablet toolbar
-STR_TABLET_X                                                    :{BLACK}X
+STR_TABLET_X                                                    :{BLACK}Trans
 STR_TABLET_TOGGLE_TRANSPARENCY_TOOLTIP                          :{BLACK}Toggle transparency
-STR_TABLET_CLOSE                                                :{BLACK}Supr
+STR_TABLET_CLOSE                                                :{BLACK}X
 STR_TABLET_CLOSE_TOOLTIP                                        :{BLACK}Close all opened windows (except pinned ones)
 STR_TABLET_SHIFT                                                :{BLACK}Shift
 STR_TABLET_SHIFT_TOOLTIP                                        :{BLACK}Press it for getting an estimated cost of executing an action
diff --git a/src/toolbar_gui.cpp b/src/toolbar_gui.cpp
index 0226d61dce..91518203e1 100644
--- a/src/toolbar_gui.cpp
+++ b/src/toolbar_gui.cpp
@@ -2296,11 +2296,11 @@ struct TabletToolbar : Window {
 
 static const NWidgetPart _nested_tablet_simple_widgets[] = {
 	NWidget(NWID_VERTICAL),
-		NWidget(WWT_PUSHTXTBTN, COLOUR_GREY, WID_TT_X), SetDataTip(STR_TABLET_X, STR_TABLET_TOGGLE_TRANSPARENCY_TOOLTIP),
 		NWidget(WWT_PUSHTXTBTN, COLOUR_GREY, WID_TT_DELETE), SetDataTip(STR_TABLET_CLOSE, STR_TABLET_CLOSE_TOOLTIP),
+		NWidget(WWT_PUSHTXTBTN, COLOUR_GREY, WID_TT_X), SetDataTip(STR_TABLET_X, STR_TABLET_TOGGLE_TRANSPARENCY_TOOLTIP),
 		NWidget(WWT_TEXTBTN, COLOUR_GREY, WID_TT_SHIFT), SetDataTip(STR_TABLET_SHIFT, STR_TABLET_SHIFT_TOOLTIP),
 		NWidget(WWT_TEXTBTN, COLOUR_GREY, WID_TT_CTRL), SetDataTip(STR_TABLET_CTRL, STR_TABLET_CTRL_TOOLTIP),
-		NWidget(WWT_TEXTBTN, COLOUR_GREY, WID_TT_MOVE), SetDataTip(STR_TABLET_MOVE, STR_TABLET_MOVE_TOOLTIP),
+		//NWidget(WWT_TEXTBTN, COLOUR_GREY, WID_TT_MOVE), SetDataTip(STR_TABLET_MOVE, STR_TABLET_MOVE_TOOLTIP),
 		EndContainer(),
 };
 
-- 
2.55.0

